src/composables/accounts.ts

```diff
@@ -373,12 +367,12 @@ export function useAccounts() {
     activeAccountGlobalIdx.value = 0;
   }
 
-  async function waitForSessionEncryptionData() {
+  async function waitForSessionEncryptionKey() {
```

Comment:
As this function is not only waiting for the encryption key but synchronises the state between background and popup I suggest to align the name to this. Maybe something like `syncBackgroundEncryptionKey` or something similar?